parking_permits_app/reversion.py

```diff
@@ -54,15 +54,16 @@ def change_message(self):
             )
             return f"{self.field.verbose_name}: {old_instance} --> {new_instance}"
         elif isinstance(self.field, models.DateTimeField):
-            format_old = self.old_value.strftime(TIME_FORMAT)
```

Comment:
I think these were in previous PR so you might need to rebase to develop

Reply:
Yeah, thought you were away, so include the change here also. Since the changed lines are identical, so I think it doesn't matter